core/src/test/scala/org/apache/spark/util/SizeEstimatorSuite.scala

```diff
@@ -135,5 +160,20 @@ class SizeEstimatorSuite
     assertResult(64)(SizeEstimator.estimate(DummyString("a")))
     assertResult(64)(SizeEstimator.estimate(DummyString("ab")))
     assertResult(72)(SizeEstimator.estimate(DummyString("abcdefgh")))
+    
```

Comment:
I didn't consider that. But after a look at the code, It seems the `ResetSystemProperties` trait already takes care of that.